canonical/canonical.com
# Rebuild travel and lifestyle page

## Done

- Applied redesign as per [design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?node-id=7367%3A20953&t=Ghb8qeVFsNnSVkNO-0)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-810.demos.haus/careers/travel
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check against redesign
- Note that the copy for this page is a combination of what were once 2 different pages: [travel](https://docs.google.com/document/d/1H2ILhAUckOWyah_6V69ImwLWOSRlXkaYnIhOHkUjclc/edit#heading=h.jyjo9jkeh22g) and [lifestyle](https://docs.google.com/document/d/15Ofb6vRUXbQ3evcWmrOu9ymuY37ayg4M_xZ2QXQyIts/edit#heading=h.jyjo9jkeh22g). So please check the final copy against how these were merged in the design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-177
